packages/arb-token-bridge-ui/package.json

```diff
@@ -30,14 +30,14 @@
     "fetch-retry": "^5.0.3",
     "graphql": "^16.8.1",
     "lodash-es": "^4.17.21",
-    "next": "^13.5.6",
+    "next": "^14.2.5",
     "next-query-params": "^5.0.0",
     "overmind": "^28.0.1",
     "overmind-react": "^29.0.1",
     "posthog-js": "^1.57.2",
     "query-string": "^8.1.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
```

Comment:
these two are also bumped based on this page:
https://nextjs.org/docs/app/building-your-application/upgrading/version-14

no other changes are needed